ui/store/actions.ts

```diff
@@ -4599,3 +4671,18 @@ export async function disableDesktop() {
   return await submitRequestToBackground('disableDesktop');
 }
 ///: END:ONLY_INCLUDE_IN
+
+///: BEGIN:ONLY_INCLUDE_IN(mmi)
+export const getMMIActions = () =>
+  mmiActionsFactory({
+    log,
+    showLoadingIndication,
+    submitRequestToBackground,
+    displayWarning,
+    hideLoadingIndication,
+    forceUpdateMetamaskState,
+    showModal,
+
+    callBackgroundMethod,
+  });
```

Comment:
can you describe what this is doing? Looks like you are passing access to our background connection to an external library which I doubt we are comfortable with.